src/sql/base/browser/ui/table/table.ts

```diff
@@ -132,6 +132,24 @@ export class Table<T extends Slick.SlickData> extends Widget implements IDisposa
 
 		this._grid.onKeyDown.subscribe((e, args: Slick.OnKeyDownEventArgs<T>) => {
 			const evt = (e as JQuery.TriggeredEvent).originalEvent as KeyboardEvent;
+			if (evt.altKey && [37, 39].includes(evt.keyCode)) {
+
+				const activeCell = this._grid.getActiveCell();
+				if (activeCell) {
+					const columns = this._grid.getColumns();
+					if (evt.keyCode === 37) {
+						if (columns[activeCell.cell].width > 10) {
+							columns[activeCell.cell].width -= 10;
```

Comment:
Why 10? Are you trying to simulate dragging the columns to change the width? (comments for intended functionality would be nice...)

Is that a common pattern you've seen elsewhere?

Reply:
Sure, you should leave a comment about that - magic numbers are bad

Was this feature something you came up with yourself or is there a standard you're following here?

Reply:
I have come up with this feature. I am not following any existing standard on it.